vendor/vendor.json

```diff
@@ -585,14 +585,22 @@
 			"path": "github.com/biogo/hts/bgzf",
 			"revision": "50da7d4131a3b5c9d063932461cab4d1fafb20b0"
 		},
+		{
+			"checksumSHA1": "7GMQgpfoSQv4QcaREE2GzSZvlQI=",
+			"path": "github.com/cheggaaa/pb",
+			"revision": "2af8bbdea9e99e83b3ac400d8f6b6d1b8cbbf338",
+			"revisionTime": "2018-05-21T09:56:06Z",
+			"version": "v1.0.25",
+			"versionExact": "v1.0.25"
+		},
 		{
 			"checksumSHA1": "X2/71FBrn4pA3WcA620ySVO0uHU=",
 			"path": "github.com/creack/goselect",
 			"revision": "528c74964609a58f7c17471525659c9b71cd499b",
 			"revisionTime": "2018-02-10T03:43:46Z"
 		},
 		{
-			"checksumSHA1": "/5cvgU+J4l7EhMXTK76KaCAfOuU=",
+			"checksumSHA1": "Lf3uUXTkKK5DJ37BxQvxO1Fq+K8=",
```

Comment:
is this change necessary? When I build locally it seems to be updating itself to a different SHA: `"checksumSHA1": "dvabztWVQX8f6oMLRyv4dLH+TGY=",`

Reply:
ok, it's from a failed revert since this was originally from like forever ago. rm'ing it.